### PR TITLE
feat: redesign viewer page

### PR DESCRIPTION
**Pull request by ryanseee**
redesigning of viewer page (page after document is verified)

### CODE REVIEW

File: src/components/AddressInfo/AddressInfo.tsx [hunk 1]
```diff
@@ -40,19 +40,23 @@ export const AddressInfoUnStyled: FunctionComponent<AddressInfoProps> = ({
...
-    ${tw`text-sm text-gray-500 font-semibold`}
+    ${tw`text-sm text-cloud-300 font-semibold`}
   }
 
-  .etherum-address {
-    display: inline-block;
-    word-break: break-all;
+  .ethereum-address {
+    ${tw`inline-block break-all text-cerulean-200`}
   }
```
Comment (simboonlong) [inline src/components/AddressInfo/AddressInfo.tsx]: shift tw classes to div

File: src/components/AddressInfo/AddressInfo.tsx [hunk 2]
```diff
@@ -40,19 +40,23 @@ export const AddressInfoUnStyled: FunctionComponent<AddressInfoProps> = ({
...
+
+  // .etherum-address {
+  //   display: inline-block;
+  //   word-break: break-all;
+  // }
```
Comment (simboonlong) [inline src/components/AddressInfo/AddressInfo.tsx]: remove

File: src/components/AssetManagementPanel/AssetManagementForm/AssetManagementForm.stories.tsx [hunk 3]
```diff
@@ -839,6 +907,8 @@ export const EndorseTransferHolderBeneficiaryPending = () => {
       onRestoreToken={() => alert("Reject document surrender")}
       restoreTokenState={FormState.UNINITIALIZED}
       tokenId=""
+      document={obfuscatedDocument}
+      verificationStatus={whenDocumentValidAndIssuedByDns as VerificationFragment[]}
     />
   );
 };
```
Comment (simboonlong) [inline src/components/AssetManagementPanel/AssetManagementForm/AssetManagementForm.stories.tsx]: think previously already missing 1 tokenBurnt story, can add that?

File: src/components/AssetManagementPanel/AssetManagementForm/FormVariants/AcceptSurrenderedForm/AcceptSurrenderedForm.tsx [hunk 4]
```diff
@@ -58,18 +58,23 @@ export const AcceptSurrenderedForm: FunctionComponent<AcceptSurrenderedFormProps
...
             </div>
           </div>
         </div>
         <div className="flex flex-wrap pb-4">
-          <div className="w-auto ml-auto">
+          {/* <div className="w-full lg:w-auto self-end">
+            <TagBorderedLg id="surrender-sign" className="text-rose border-none font-ubuntu pt-0 pb-0 pl-0 pr-0">
+              Surrendered To Issuer
+            </TagBorderedLg>
+          </div> */}
```
Comment (simboonlong) [inline src/components/AssetManagementPanel/AssetManagementForm/FormVariants/AcceptSurrenderedForm/AcceptSurrenderedForm.tsx]: remove?

File: src/components/AssetManagementPanel/AssetManagementForm/FormVariants/AcceptSurrenderedForm/AcceptSurrenderedForm.tsx [hunk 5]
```diff
@@ -79,15 +84,15 @@ export const AcceptSurrenderedForm: FunctionComponent<AcceptSurrenderedFormProps
...
+                  className="bg-rose text-white rounded-xl text-lg py-2 px-3 shadow-none hover:bg-rose-400"
                   onClick={handleDestroyToken}
                   disabled={isDestroyTokenPendingConfirmation}
                   data-testid={"acceptSurrenderBtn"}
                 >
                   {isDestroyTokenPendingConfirmation ? (
                     <LoaderSpinner data-testid={"accept-loader"} />
                   ) : (
-                    <>Shred Document</>
+                    <>Accept Document</>
```
Comment (simboonlong) [inline src/components/AssetManagementPanel/AssetManagementForm/FormVariants/AcceptSurrenderedForm/AcceptSurrenderedForm.tsx]: text wise, no longer shred?

File: src/components/AssetManagementPanel/AssetManagementForm/FormVariants/ActionSelectionForm/ActionSelectionForm.tsx [hunk 6]
```diff
@@ -114,6 +126,13 @@ export const ActionSelectionForm: FunctionComponent<ActionSelectionFormProps> =
       </div>
       {!isTokenBurnt && (
         <div className="flex flex-wrap pb-4">
+          {/* {isSurrendered && (
+            <div className="w-full lg:w-auto self-end">
+              <TagBorderedLg id="surrender-sign" className="text-rose border-none font-ubuntu pt-0 pb-0 pl-0 pr-0">
+                Surrendered To Issuer
+              </TagBorderedLg>
+            </div>
+          )} */}
```
Comment (simboonlong) [inline src/components/AssetManagementPanel/AssetManagementForm/FormVariants/ActionSelectionForm/ActionSelectionForm.tsx]: remove?

File: src/components/AssetManagementPanel/AssetManagementForm/FormVariants/RejectSurrenderedForm/RejectSurrenderedForm.tsx [hunk 7]
```diff
@@ -84,18 +84,23 @@ export const RejectSurrenderedForm: FunctionComponent<RejectSurrenderedFormProps
...
             </div>
           </div>
         </div>
         <div className="flex flex-wrap pb-4">
-          <div className="w-auto ml-auto">
+          {/* <div className="w-full lg:w-auto self-end">
+            <TagBorderedLg id="surrender-sign" className="text-rose border-none font-ubuntu pt-0 pb-0 pl-0 pr-0">
+              Surrendered To Issuer
+            </TagBorderedLg>
+          </div> */}
```
Comment (simboonlong) [inline src/components/AssetManagementPanel/AssetManagementForm/FormVariants/RejectSurrenderedForm/RejectSurrenderedForm.tsx]: remove?

File: src/components/CertificateViewer.tsx [hunk 8]
```diff
@@ -2,6 +2,7 @@ import { VerificationFragment } from "@govtechsg/oa-verify";
 import { getData, utils, v2, WrappedDocument } from "@govtechsg/open-attestation";
 import React, { FunctionComponent, useCallback, useEffect, useState } from "react";
 import { useDispatch } from "react-redux";
+// import { Link } from "react-router-dom";
 import { useTokenInformationContext } from "../common/contexts/TokenInformationContext";
```
Comment (simboonlong) [inline src/components/CertificateViewer.tsx]: remove

File: src/components/CertificateViewer.tsx [hunk 9]
```diff
@@ -82,8 +83,8 @@ export const CertificateViewer: FunctionComponent<CertificateViewerProps> = ({ d
 
   const renderedEndorsementChain = (
     <div className="bg-cerulean-50 no-print">
-      <DocumentStatus verificationStatus={verificationStatus} />
-      <ObfuscatedMessage document={document} />
+      {/* <DocumentStatus verificationStatus={verificationStatus} />
+      <ObfuscatedMessage document={document} /> */}
       {tokenRegistryAddress && (
```
Comment (simboonlong) [inline src/components/CertificateViewer.tsx]: remove

File: src/components/CertificateViewer.tsx [hunk 10]
```diff
@@ -96,18 +97,39 @@ export const CertificateViewer: FunctionComponent<CertificateViewerProps> = ({ d
...
+          </h3>
+          <Link
+            to="/contact"
+            className="h-12 px-4 py-2 ml-auto my-auto rounded-xl text-white bg-tangerine hover:bg-tangerine-600 hover:text-gray-200"
+            data-testid="try-demo"
+          >
+            <h3 className="font-normal text-2xl">Try our demo now</h3>
+          </Link>
+        </div>
+      </div> */}
```
Comment (simboonlong) [inline src/components/CertificateViewer.tsx]: remove

File: src/components/DecentralisedTemplateRenderer/MultiTabs.tsx [hunk 11]
```diff
@@ -23,10 +23,13 @@ export const MultiTabs: FunctionComponent<MultiTabsProps> = ({
 }) => {
   return (
     <div className="container">
-      <div className="flex flex-wrap items-end">
+      {/* <div className="flex flex-wrap items-end"> */}
+      <div className="flex overflow-x-auto items-end">
```
Comment (simboonlong) [inline src/components/DecentralisedTemplateRenderer/MultiTabs.tsx]: remove

File: src/components/DocumentStatus/DocumentStatus.tsx [hunk 12]
```diff
@@ -79,7 +77,7 @@ const DocumentStatusStyles = styled.div`
   }
 
   .issuedby {
-    ${tw`text-gray-700 text-lg font-semibold`}
+    ${tw`text-cloud-900 text-lg font-semibold`}
```
Comment (simboonlong) [inline src/components/DocumentStatus/DocumentStatus.tsx]: can shift

File: src/pages/settings/settings.tsx [hunk 13]
```diff
@@ -14,7 +33,16 @@ export const SettingsPage: FunctionComponent = () => (
...
+          <div className="mr-4 mb-4" key={index}>
+            <Link to={details.pathLink} className="inline-block">
+              <TileInfo title={details.title} description={details.description} tileIcon={details.tileIcon} />
+            </Link>
+          </div>
+        ))}
+      </div>
+      {/* <div className="mt-4">
         <Link to="/settings/address-book" className="inline-block mb-4">
           <TileInfo
```
Comment (simboonlong) [inline src/pages/settings/settings.tsx]: remove

File: src/components/AddressInfo/AddressInfo.tsx (hunk 1 above)
Comment (ryanseee) [reply src/components/AddressInfo/AddressInfo.tsx]: done.

File: src/components/AddressInfo/AddressInfo.tsx (hunk 2 above)
Comment (ryanseee) [reply src/components/AddressInfo/AddressInfo.tsx]: done.

File: src/components/AssetManagementPanel/AssetManagementForm/AssetManagementForm.stories.tsx (hunk 3 above)
Comment (ryanseee) [reply src/components/AssetManagementPanel/AssetManagementForm/AssetManagementForm.stories.tsx]: done.

File: src/components/AssetManagementPanel/AssetManagementForm/FormVariants/AcceptSurrenderedForm/AcceptSurrenderedForm.tsx (hunk 4 above)
Comment (ryanseee) [reply src/components/AssetManagementPanel/AssetManagementForm/FormVariants/AcceptSurrenderedForm/AcceptSurrenderedForm.tsx]: done.

File: src/components/AssetManagementPanel/AssetManagementForm/FormVariants/AcceptSurrenderedForm/AcceptSurrenderedForm.tsx (hunk 5 above)
Comment (ryanseee) [reply src/components/AssetManagementPanel/AssetManagementForm/FormVariants/AcceptSurrenderedForm/AcceptSurrenderedForm.tsx]: reverted back to Shred Document after discussion.

File: src/components/AssetManagementPanel/AssetManagementForm/FormVariants/ActionSelectionForm/ActionSelectionForm.tsx (hunk 6 above)
Comment (ryanseee) [reply src/components/AssetManagementPanel/AssetManagementForm/FormVariants/ActionSelectionForm/ActionSelectionForm.tsx]: done.

File: src/components/CertificateViewer.tsx (hunk 8 above)
Comment (ryanseee) [reply src/components/CertificateViewer.tsx]: done.

File: src/components/AssetManagementPanel/AssetManagementForm/FormVariants/RejectSurrenderedForm/RejectSurrenderedForm.tsx (hunk 7 above)
Comment (ryanseee) [reply src/components/AssetManagementPanel/AssetManagementForm/FormVariants/RejectSurrenderedForm/RejectSurrenderedForm.tsx]: done.

File: src/components/CertificateViewer.tsx (hunk 9 above)
Comment (ryanseee) [reply src/components/CertificateViewer.tsx]: done.

File: src/components/CertificateViewer.tsx (hunk 10 above)
Comment (ryanseee) [reply src/components/CertificateViewer.tsx]: done.

File: src/components/DecentralisedTemplateRenderer/MultiTabs.tsx (hunk 11 above)
Comment (ryanseee) [reply src/components/DecentralisedTemplateRenderer/MultiTabs.tsx]: done.

File: src/components/DocumentStatus/DocumentStatus.tsx (hunk 12 above)
Comment (ryanseee) [reply src/components/DocumentStatus/DocumentStatus.tsx]: done, shifted tw to div in some of the other files too.

File: src/pages/settings/settings.tsx (hunk 13 above)
Comment (ryanseee) [reply src/pages/settings/settings.tsx]: done.

File: src/components/CertificateViewer.tsx [hunk 14]
```diff
@@ -96,18 +94,24 @@ export const CertificateViewer: FunctionComponent<CertificateViewerProps> = ({ d
...
+            document={document}
+            verificationStatus={verificationStatus}
           />
+        ) : (
+          <div className="container">
+            <DocumentStatus verificationStatus={verificationStatus} />
+            <ObfuscatedMessage document={document} />
+          </div>
         )}
       </div>
```
Comment (simboonlong) [inline src/components/CertificateViewer.tsx]: um any better way at managing state? this is affecting alot of files

File: src/components/CertificateViewer.tsx (hunk 8 above)
Comment (simboonlong) [inline src/components/CertificateViewer.tsx]: can remove

**Review by simboonlong — APPROVED** [review]
lgtm

**Pull request merged**